tests/unit/package_managers/test_generic.py

```diff
@@ -207,6 +207,61 @@ def test_resolve_generic_lockfile_invalid(
     assert expected_err in str(exc_info.value)
 
 
+@pytest.mark.parametrize(
+    ["lockfile", "expected"],
```

Comment:
```suggestion
    ["lockfile_content", "expected_components"]
```
nitpick: I would choose slightly more verbose names.

Reply:
Changed.